client/src/components/CarouselDots/CarouselDots.js

```diff
@@ -0,0 +1,124 @@
+
+import React, {Component} from 'react';
+import "./CarouselDots.css";
+import ReactDOM from "react-dom";
+
+// Class Based React Component
+class CarouselDots extends Component{
+  constructor(props){
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "CarouselDots",
+      numberOfDots: props.numberOfDots,
+      activeLocation: props.activeLocation,
+      activeDotColor: props.activeDotColor,
+      inactiveDotColor: props.inactiveDotColor,
+      hasBorder: props.hasBorder,
+      borderWeight: props.borderWeight,
+      height: 50,
+      width: 50
+
+    };
+  }
+
+
+  // Runs after Component is loaded in the broswer
+  componentDidMount() {
+    let height = ReactDOM.findDOMNode(this).parentNode.offsetHeight;
+    this.setState({
+      height: height,
+      width: height
+    })
+  }
```

Comment:
It's unclear to me why we need this and it seems a little dirty. What is this doing and is there another way to achieve this?

For instance, the height and width can probably just be fixed in the CSS